k8s/openebs-operator.yaml

```diff
@@ -156,6 +156,13 @@ spec:
           value: "openebs/cstor-volume-mgmt:ci"
         - name: OPENEBS_IO_VOLUME_MONITOR_IMAGE
           value: "openebs/m-exporter:ci"
+        livenessProbe:
```

Comment:
Separately, can we use the same ExecAction to add readinessProbe?

Reply:
Yes. It does help to add a readinessProbe to maya-apiserver deployment, since it is frontended by a service.